src/routes/game/GameView.vue

```diff
@@ -181,28 +181,7 @@
               </div>
             </template>
           </v-card>
-          <ScrapDialog :scrap="scrap">
-            <template #activator>
-              <div id="scrap" class="d-flex flex-column align-center">
-                <Transition :name="threesTransition">
-                  <GameCard
-                    v-if="showScrapChoice"
-                    :suit="gameStore.lastEventCardChosen.suit"
-                    :rank="gameStore.lastEventCardChosen.rank"
-                    class="gameCard"
-                    data-cy="scrap-chosen-card"
-                  />
-                  <div v-else class="d-flex flex-column align-center scrapPile">
-                    <h3>{{ $t('game.scrap') }}</h3>
-                    <span>({{ scrap.length }})</span>
-                    <v-btn variant="outlined" color="primary" class="mt-4">
-                      {{ $t('game.view') }}
-                    </v-btn>
-                  </div>
-                </Transition>
-              </div>
-            </template>
-          </ScrapDialog>
+          <ScrapDialog :scrap="scrap" />
```

Comment:
Pulled all the layout stuff directly into the ScrapDialog itself. This helps slim down GameView and consolidates the styles and functionality in that one component